lblod/ember-rdfa-editor-lblod-plugins
# GN-4897: ensure older decision articles can be correctly parsed into the new `structure` nodes

### Overview
This PR includes the following changes:
- A fix for the structure number serialization
- Disabling the editable title for structures representing decision articles
- Addition of a backwards-compatible parsing-rule to the `structure` node-spec, responsible for parsing decision articles in the old format.

##### connected issues and PRs:
[GN-4897](https://binnenland.atlassian.net/browse/GN-4897)

### How to test/reproduce
- Open the dummy app
- Insert a sample decision template (which at the moment still contains decision articles in the old format)
- Ensure the decision (include the articles in the new format) are correctly parsed
- When adding new article structures, the numbering should still hold up

### Challenges/uncertainties
Note: the backwards compatible parsing-rule throws away the RDFa it parses from the old articles. As the old articles were not yet used in an rdfa-aware way, this should not be a problem.

### Checks PR readiness
